remote/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@gitpod/gitpod-protocol": "main",
-    "@gitpod/ide-metrics-api-grpcweb": "ak-ext-metrics",
+    "@gitpod/ide-metrics-api-grpcweb": "^0.0.1-main.4783",
```

Comment:
Global search with `"@gitpod/ide-metrics-api-grpcweb": "ak-ext-metrics"` and found it. Why we need this dependency in the `remote` and `remote/web` folders?

cc @akosyakov